atlassian/bitbucket/cloud/repositories/defaultReviewers.py

```diff
@@ -70,31 +71,15 @@ def get(self, user):
         return default_reviewer
 
 
-class DefaultReviewer(BitbucketCloudBase):
+class DefaultReviewer(User):
     def __init__(self, url, data, *args, **kwargs):
-        super(DefaultReviewer, self).__init__(url, *args, data=data, expected_type="user", **kwargs)
-
-    @property
-    def display_name(self):
-        return str(self.get_data("display_name"))
-
-    @property
-    def nickname(self):
-        return self.get_data("nickname")
-
-    @property
-    def account_id(self):
-        return self.get_data("account_id")
-
-    @property
-    def uuid(self):
-        return self.get_data("uuid")
+        super(DefaultReviewer, self).__init__(url, data, *args, **kwargs)
 
     def delete(self):
         """
         Deletes the default reviewer
         """
-        data = super(DefaultReviewer, self).delete(None)
+        data = super(User, self).delete(None)
         if "errors" in data:
             return
-        return DefaultReviewer(self.url, data, **self._new_session_args)
+        return User(self.url, data, **self._new_session_args)
```

Comment:
TODO: DefaultReviewer

Reply:
Yes. Because the user is deleted, the returned object should be empty. But what about `if "errors"...` , IMO this should raise an exception?

Reply:
This if block is copied from the old non OO implementation. If the user activates the raw responses a error response will not trigger an exception. In the OO interface the raw response shall never be activated by the user and therefore a exception is thrown. Maybe it can be removed.

Reply:
Ok. I think that's no task from this implementation. Therefore it should be clarified / modified in a extra pull request.